packages/truffle-box/test/box.js

```diff
@@ -1,6 +1,6 @@
 const path = require("path");
-const regularFs = require("fs");
-const fs = require("fs-extra");
+const fs = require("fs");
+const fse = require("fs-extra");
```

Comment:
`fs-extra` contains all the methods of `fs` so we won't need to use `fs` if we use `fs-extra`. Also, all the async methods return promises if you don't provide a callback.

Reply:
ah, `fs` is used in here as a stub since `Box.checkDir` uses `fs` and not `fse`.

Makes sense to simplify.